configuration.go

```diff
@@ -0,0 +1,71 @@
+package corbado
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/corbado/corbado-go/pkg/assert"
+	"github.com/corbado/corbado-go/pkg/generated/api"
+)
+
+type Configuration struct {
```

Comment:
I'm not sure if it's idiomatic to use `Configuration` instead of just `Config` in to. Maybe worth checking some other repos. I personally like `Config` better

Reply:
changed it back to Config, my 2024 present for you :)